compass/landice/tests/greenland/mesh.py

```diff
@@ -56,136 +59,67 @@ def run(self):
-        logger.info('calling interpolate_to_mpasli_grid.py')
-        args = ['interpolate_to_mpasli_grid.py', '-s',
-                data_path + '/BedMachineGreenland-v5_edits_floodFill_extrap.nc',  # noqa
-                '-d', 'GIS.nc', '-m', 'e',
-                '-w', 'BedMachine_to_MPAS_weights.nc']
-        check_call(args, logger=logger)
-
-        logger.info('calling interpolate_to_mpasli_grid.py')
-        args = ['interpolate_to_mpasli_grid.py', '-s',
-                data_path + '/greenland_vel_mosaic500_extrap.nc',
-                '-d', 'GIS.nc', '-m', 'e',
-                '-w', 'measures_to_MPAS_weights.nc',
-                '-v', 'observedSurfaceVelocityX',
-                'observedSurfaceVelocityY',
-                'observedSurfaceVelocityUncertainty']
-        check_call(args, logger=logger)
-
-        logger.info('Marking domain boundaries dirichlet')
-        args = ['mark_domain_boundaries_dirichlet.py',
-                '-f', 'GIS.nc']
-        check_call(args, logger=logger)
-
+        dst_scrip_file = f"{self.mesh_filename.split('.')[:-1][0]}_scrip.nc"
+        scrip_from_mpas(self.mesh_filename, dst_scrip_file)
+
+        # Now perform bespoke interpolation of geometry and velocity data
+        # from their respective sources
+        interp_gridded2mali(self, bedmachine_dataset, dst_scrip_file, nProcs,
+                            self.mesh_filename, src_proj, variables="all")
+
+        # only interpolate a subset of MEASURES varibles onto the MALI mesh
+        measures_vars = ['observedSurfaceVelocityX',
+                         'observedSurfaceVelocityY',
+                         'observedSurfaceVelocityUncertainty']
+        interp_gridded2mali(self, measures_dataset, dst_scrip_file, nProcs,
+                            self.mesh_filename, src_proj,
+                            variables=measures_vars)
+
+        # perform some final cleanup details
+        clean_up_after_interp(self.mesh_filename)
+
+        # create graph file
         logger.info('creating graph.info')
-        make_graph_file(mesh_filename=mesh_name,
+        make_graph_file(mesh_filename=self.mesh_filename,
                         graph_filename='graph.info')
-        # Create a backup in case clean-up goes awry
-        copyfile('GIS.nc', 'GIS_backup.nc')
-
-        # Clean up: trim to iceMask and set large velocity
-        # uncertainties where appropriate.
-        data = netCDF4.Dataset('GIS.nc', 'r+')
-        data.set_auto_mask(False)
-        data.variables['thickness'][:] *= (data.variables['iceMask'][:] > 1.5)
-
-        mask = np.logical_or(
-            np.isnan(
-                data.variables['observedSurfaceVelocityUncertainty'][:]),
-            data.variables['thickness'][:] < 1.0)
-        mask = np.logical_or(
-            mask,
-            data.variables['observedSurfaceVelocityUncertainty'][:] == 0.0)
-        data.variables['observedSurfaceVelocityUncertainty'][0, mask[0, :]] = 1.0  # noqa
```

Comment:
This was replaced by the cal to `compass.landice.mesh.clean_up_after_interp`. Given the code here and that within the function are basically identical, I'm assuming that's OK.